wandb: Currently logged in as: hadars (autoconvnext). Use `wandb login --relogin` to force relogin
wandb: wandb version 0.13.5 is available!  To upgrade, please run:
wandb:  $ pip install wandb --upgrade
wandb: Tracking run with wandb version 0.13.4
wandb: Run data is saved locally in /home/s3092593/benchmarking_rl/wandb/run-20221201_042716-2penmxri
wandb: Run `wandb offline` to turn off syncing.
wandb: Syncing run electric-wood-1
wandb: ⭐️ View project at https://wandb.ai/autoconvnext/benchmark_final_impala_large_1_DQN_modern_Phoenix_model_37250000_0_impala_large_1
wandb: 🚀 View run at https://wandb.ai/autoconvnext/benchmark_final_impala_large_1_DQN_modern_Phoenix_model_37250000_0_impala_large_1/runs/2penmxri
wandb: Waiting for W&B process to finish... (success).
wandb: 
wandb: Run history:
wandb: loss █▆▆▅▄▄▃▃▃▃▃▂▂▂▂▂▂▂▂▂▂▂▂▁▁▁▁▁▁▁
wandb:  mae █▇▆▅▄▄▃▃▂▂▂▂▂▂▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁
wandb: 
wandb: Run summary:
wandb: loss 0.22478
wandb:  mae 0.94557
wandb: 
wandb: Synced electric-wood-1: https://wandb.ai/autoconvnext/benchmark_final_impala_large_1_DQN_modern_Phoenix_model_37250000_0_impala_large_1/runs/2penmxri
wandb: Synced 6 W&B file(s), 0 media file(s), 1 artifact file(s) and 0 other file(s)
wandb: Find logs at: ./wandb/run-20221201_042716-2penmxri/logs
wandb: wandb version 0.13.5 is available!  To upgrade, please run:
wandb:  $ pip install wandb --upgrade
wandb: Tracking run with wandb version 0.13.4
wandb: Run data is saved locally in /home/s3092593/benchmarking_rl/wandb/run-20221201_053742-29hwka9e
wandb: Run `wandb offline` to turn off syncing.
wandb: Syncing run fancy-valley-2
wandb: ⭐️ View project at https://wandb.ai/autoconvnext/benchmark_final_impala_large_1_DQN_modern_Phoenix_model_37250000_0_impala_large_1
wandb: 🚀 View run at https://wandb.ai/autoconvnext/benchmark_final_impala_large_1_DQN_modern_Phoenix_model_37250000_0_impala_large_1/runs/29hwka9e
wandb: Waiting for W&B process to finish... (success).
wandb: 
wandb: Run history:
wandb: loss █▄▄▃▃▃▃▃▂▂▂▂▂▂▂▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁
wandb:  mae █▅▄▄▃▃▂▂▂▂▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▂
wandb: 
wandb: Run summary:
wandb: loss 0.23011
wandb:  mae 0.96356
wandb: 
wandb: Synced fancy-valley-2: https://wandb.ai/autoconvnext/benchmark_final_impala_large_1_DQN_modern_Phoenix_model_37250000_0_impala_large_1/runs/29hwka9e
wandb: Synced 6 W&B file(s), 0 media file(s), 1 artifact file(s) and 0 other file(s)
wandb: Find logs at: ./wandb/run-20221201_053742-29hwka9e/logs
wandb: wandb version 0.13.5 is available!  To upgrade, please run:
wandb:  $ pip install wandb --upgrade
wandb: Tracking run with wandb version 0.13.4
wandb: Run data is saved locally in /home/s3092593/benchmarking_rl/wandb/run-20221201_064831-ichh5ggl
wandb: Run `wandb offline` to turn off syncing.
wandb: Syncing run devoted-mountain-3
wandb: ⭐️ View project at https://wandb.ai/autoconvnext/benchmark_final_impala_large_1_DQN_modern_Phoenix_model_37250000_0_impala_large_1
wandb: 🚀 View run at https://wandb.ai/autoconvnext/benchmark_final_impala_large_1_DQN_modern_Phoenix_model_37250000_0_impala_large_1/runs/ichh5ggl
wandb: Waiting for W&B process to finish... (success).
wandb: 
wandb: Run history:
wandb: loss █▄▄▃▃▂▂▂▂▂▂▂▂▂▂▂▂▁▂▁▁▁▁▁▁▁▁▁▁▁
wandb:  mae █▅▅▄▃▃▃▂▂▂▂▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁
wandb: 
wandb: Run summary:
wandb: loss 0.22221
wandb:  mae 0.95681
wandb: 
wandb: Synced devoted-mountain-3: https://wandb.ai/autoconvnext/benchmark_final_impala_large_1_DQN_modern_Phoenix_model_37250000_0_impala_large_1/runs/ichh5ggl
wandb: Synced 6 W&B file(s), 0 media file(s), 1 artifact file(s) and 0 other file(s)
wandb: Find logs at: ./wandb/run-20221201_064831-ichh5ggl/logs
wandb: wandb version 0.13.5 is available!  To upgrade, please run:
wandb:  $ pip install wandb --upgrade
wandb: Tracking run with wandb version 0.13.4
wandb: Run data is saved locally in /home/s3092593/benchmarking_rl/wandb/run-20221201_075921-2ihtyc33
wandb: Run `wandb offline` to turn off syncing.
wandb: Syncing run ancient-microwave-4
wandb: ⭐️ View project at https://wandb.ai/autoconvnext/benchmark_final_impala_large_1_DQN_modern_Phoenix_model_37250000_0_impala_large_1
wandb: 🚀 View run at https://wandb.ai/autoconvnext/benchmark_final_impala_large_1_DQN_modern_Phoenix_model_37250000_0_impala_large_1/runs/2ihtyc33
wandb: Waiting for W&B process to finish... (success).
wandb: 
wandb: Run history:
wandb: loss █▄▄▄▃▃▃▃▃▃▃▃▂▂▂▂▁▁▁▁▁▁▁▁▁▁▁▁▁▁
wandb:  mae █▅▄▄▃▃▂▂▂▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▂▂▂▂▂▂
wandb: 
wandb: Run summary:
wandb: loss 0.22247
wandb:  mae 0.99178
wandb: 
wandb: Synced ancient-microwave-4: https://wandb.ai/autoconvnext/benchmark_final_impala_large_1_DQN_modern_Phoenix_model_37250000_0_impala_large_1/runs/2ihtyc33
wandb: Synced 6 W&B file(s), 0 media file(s), 1 artifact file(s) and 0 other file(s)
wandb: Find logs at: ./wandb/run-20221201_075921-2ihtyc33/logs
wandb: wandb version 0.13.5 is available!  To upgrade, please run:
wandb:  $ pip install wandb --upgrade
wandb: Tracking run with wandb version 0.13.4
wandb: Run data is saved locally in /home/s3092593/benchmarking_rl/wandb/run-20221201_090942-1yc9e280
wandb: Run `wandb offline` to turn off syncing.
wandb: Syncing run jolly-capybara-5
wandb: ⭐️ View project at https://wandb.ai/autoconvnext/benchmark_final_impala_large_1_DQN_modern_Phoenix_model_37250000_0_impala_large_1
wandb: 🚀 View run at https://wandb.ai/autoconvnext/benchmark_final_impala_large_1_DQN_modern_Phoenix_model_37250000_0_impala_large_1/runs/1yc9e280
wandb: Waiting for W&B process to finish... (success).
wandb: 
wandb: Run history:
wandb: loss █▄▄▄▃▃▃▃▃▂▂▂▂▂▂▂▂▂▂▂▂▁▁▁▁▁▁▁▁▁
wandb:  mae █▅▅▄▃▂▂▂▂▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▂▂▃
wandb: 
wandb: Run summary:
wandb: loss 0.19111
wandb:  mae 1.02122
wandb: 
wandb: Synced jolly-capybara-5: https://wandb.ai/autoconvnext/benchmark_final_impala_large_1_DQN_modern_Phoenix_model_37250000_0_impala_large_1/runs/1yc9e280
wandb: Synced 6 W&B file(s), 0 media file(s), 1 artifact file(s) and 0 other file(s)
wandb: Find logs at: ./wandb/run-20221201_090942-1yc9e280/logs
wandb: wandb version 0.13.5 is available!  To upgrade, please run:
wandb:  $ pip install wandb --upgrade
wandb: Tracking run with wandb version 0.13.4
wandb: Run data is saved locally in /home/s3092593/benchmarking_rl/wandb/run-20221201_101928-2idbsypl
wandb: Run `wandb offline` to turn off syncing.
wandb: Syncing run visionary-dragon-6
wandb: ⭐️ View project at https://wandb.ai/autoconvnext/benchmark_final_impala_large_1_DQN_modern_Phoenix_model_37250000_0_impala_large_1
wandb: 🚀 View run at https://wandb.ai/autoconvnext/benchmark_final_impala_large_1_DQN_modern_Phoenix_model_37250000_0_impala_large_1/runs/2idbsypl
wandb: Waiting for W&B process to finish... (success).
wandb: 
wandb: Run history:
wandb: loss █▄▄▄▃▃▃▂▂▂▂▂▂▂▂▂▁▁▁▁▁▁▁▁▁▁▁▁▁▁
wandb:  mae █▅▅▄▃▃▂▂▁▁▁▁▁▁▁▁▁▂▂▃▂▁▁▂▁▁▁▁▁▁
wandb: 
wandb: Run summary:
wandb: loss 0.23205
wandb:  mae 0.95879
wandb: 
wandb: Synced visionary-dragon-6: https://wandb.ai/autoconvnext/benchmark_final_impala_large_1_DQN_modern_Phoenix_model_37250000_0_impala_large_1/runs/2idbsypl
wandb: Synced 6 W&B file(s), 0 media file(s), 1 artifact file(s) and 0 other file(s)
wandb: Find logs at: ./wandb/run-20221201_101928-2idbsypl/logs
wandb: wandb version 0.13.5 is available!  To upgrade, please run:
wandb:  $ pip install wandb --upgrade
wandb: Tracking run with wandb version 0.13.4
wandb: Run data is saved locally in /home/s3092593/benchmarking_rl/wandb/run-20221201_113025-19u6zfp1
wandb: Run `wandb offline` to turn off syncing.
wandb: Syncing run dark-violet-7
wandb: ⭐️ View project at https://wandb.ai/autoconvnext/benchmark_final_impala_large_1_DQN_modern_Phoenix_model_37250000_0_impala_large_1
wandb: 🚀 View run at https://wandb.ai/autoconvnext/benchmark_final_impala_large_1_DQN_modern_Phoenix_model_37250000_0_impala_large_1/runs/19u6zfp1
--- Logging error ---
Traceback (most recent call last):
  File "/data1/s3092593/rlenv/lib/python3.8/logging/__init__.py", line 1089, in emit
    self.flush()
  File "/data1/s3092593/rlenv/lib/python3.8/logging/__init__.py", line 1069, in flush
    self.stream.flush()
OSError: [Errno 122] Disk quota exceeded
Call stack:
  File "/data1/s3092593/rlenv/lib/python3.8/threading.py", line 890, in _bootstrap
    self._bootstrap_inner()
  File "/data1/s3092593/rlenv/lib/python3.8/threading.py", line 932, in _bootstrap_inner
    self.run()
  File "/data1/s3092593/rlenv/lib/python3.8/site-packages/wandb/sdk/internal/internal_util.py", line 50, in run
    self._run()
  File "/data1/s3092593/rlenv/lib/python3.8/site-packages/wandb/sdk/internal/internal_util.py", line 101, in _run
    self._process(record)
  File "/data1/s3092593/rlenv/lib/python3.8/site-packages/wandb/sdk/internal/internal.py", line 263, in _process
    self._hm.handle(record)
  File "/data1/s3092593/rlenv/lib/python3.8/site-packages/wandb/sdk/internal/handler.py", line 130, in handle
    handler(record)
  File "/data1/s3092593/rlenv/lib/python3.8/site-packages/wandb/sdk/internal/handler.py", line 138, in handle_request
    logger.debug(f"handle_request: {request_type}")
Message: 'handle_request: stop_status'
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/data1/s3092593/rlenv/lib/python3.8/logging/__init__.py", line 1089, in emit
    self.flush()
  File "/data1/s3092593/rlenv/lib/python3.8/logging/__init__.py", line 1069, in flush
    self.stream.flush()
OSError: [Errno 122] Disk quota exceeded
Call stack:
  File "/data1/s3092593/rlenv/lib/python3.8/threading.py", line 890, in _bootstrap
    self._bootstrap_inner()
  File "/data1/s3092593/rlenv/lib/python3.8/threading.py", line 932, in _bootstrap_inner
    self.run()
  File "/data1/s3092593/rlenv/lib/python3.8/site-packages/wandb/vendor/watchdog/observers/api.py", line 199, in run
    self.dispatch_events(self.event_queue, self.timeout)
  File "/data1/s3092593/rlenv/lib/python3.8/site-packages/wandb/vendor/watchdog/observers/api.py", line 368, in dispatch_events
    handler.dispatch(event)
  File "/data1/s3092593/rlenv/lib/python3.8/site-packages/wandb/vendor/watchdog/events.py", line 454, in dispatch
    _method_map[event_type](event)
  File "/data1/s3092593/rlenv/lib/python3.8/site-packages/wandb/filesync/dir_watcher.py", line 292, in _on_file_modified
    logger.info(f"file/dir modified: { event.src_path}")
Message: 'file/dir modified: /home/s3092593/benchmarking_rl/wandb/run-20221201_113025-19u6zfp1/files/output.log'
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/data1/s3092593/rlenv/lib/python3.8/logging/__init__.py", line 1089, in emit
    self.flush()
  File "/data1/s3092593/rlenv/lib/python3.8/logging/__init__.py", line 1069, in flush
    self.stream.flush()
OSError: [Errno 122] Disk quota exceeded
Call stack:
  File "/data1/s3092593/rlenv/lib/python3.8/threading.py", line 890, in _bootstrap
    self._bootstrap_inner()
  File "/data1/s3092593/rlenv/lib/python3.8/threading.py", line 932, in _bootstrap_inner
    self.run()
  File "/data1/s3092593/rlenv/lib/python3.8/site-packages/wandb/sdk/internal/internal_util.py", line 50, in run
    self._run()
  File "/data1/s3092593/rlenv/lib/python3.8/site-packages/wandb/sdk/internal/internal_util.py", line 101, in _run
    self._process(record)
  File "/data1/s3092593/rlenv/lib/python3.8/site-packages/wandb/sdk/internal/internal.py", line 263, in _process
    self._hm.handle(record)
  File "/data1/s3092593/rlenv/lib/python3.8/site-packages/wandb/sdk/internal/handler.py", line 130, in handle
    handler(record)
  File "/data1/s3092593/rlenv/lib/python3.8/site-packages/wa